hana/install_pydbapi.sls

```diff
@@ -16,10 +16,20 @@ hana_install_python_pip:
         interval: 15
     - resolve_capabilities: true
 
+# The software_folders is retrieved in this order: 
+# 1. node.exporter.hana_client_path
+# 2. node.install.software_path
+# 3. hana.software_path
+{% if node.exporter.hana_client_path is defined or node.install.software_path is defined  %}
```

Comment:
Don't forget the `hana.software_path`. Maybe this works:

```
{% set software_path = node.exporter.hana_client_path|default(node.install.software_path)|default(hana.software_path)|default(hana.hdbserver_extract_dir) %}
```

Reply:
hdbserver_extract_dir will contain the HANA Client package so i will  skip adding that